src/Models/ChannelModel.php

```diff
@@ -152,11 +152,14 @@ public function getChannelsPinnedPosts($channelId)
     /**
      * @param $teamId
      * @param $channelName
+     * @param $include_deleted
      * @return ResponseInterface
      */
-    public function getChannelByName($teamId, $channelName)
+    public function getChannelByName($teamId, $channelName, $include_deleted = false)

```

Comment:
Hi @matak many thanks for your contribution!

May I ask you to change the method signature in `public function getChannelByName($teamId, $channelName, array $requestOptions = [])`? 

The standard chosen in this project is to let prepare the query params outside the client. 

You can take [this signature](https://github.com/gnello/php-mattermost-driver/blob/master/src/Models/ChannelModel.php#L64) as an example.

Thanks!

Reply:
I've changed the signature as proposed and merged the PR.

Thanks for your contribution!